app/assets/javascripts/discourse/app/components/about-page.gjs

```diff
@@ -230,14 +230,29 @@ export default class AboutPage extends Component {
             <AboutPageUsers @users={{@model.admins}} @truncateAt={{6}} />
           </section>
         {{/if}}
+        <span>
```

Comment:
I'd skip those span wrappers. They're almost always are incorrect html in the existing cases (i.e. they contain block content, but themselves are inline elements) so we try to remove them (and not add new ones)

Reply:
Fair enough, I've removed them.